packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -0,0 +1,490 @@
+import { store as blockEditorStore } from '@wordpress/block-editor';
+import { removep } from '@wordpress/autop';
+
+/**
+ * Internal dependencies
+ */
+import { sanitizeCommentString } from './utils';
+
+/**
+ * Renders the Comments component.
+ *
+ * @param {Object}   props                  - The component props.
+ * @param {Array}    props.threads          - The array of comment threads.
+ * @param {Function} props.onEditComment    - The function to handle comment editing.
+ * @param {Function} props.onAddReply       - The function to add a reply to a comment.
+ * @param {Function} props.onCommentDelete  - The function to delete a comment.
+ * @param {Function} props.onCommentResolve - The function to mark a comment as resolved.
+ * @return {JSX.Element} The rendered Comments component.
+ */
+export function Comments( {
+	threads,
+	onEditComment,
+	onAddReply,
+	onCommentDelete,
+	onCommentResolve,
+} ) {
+	const [ actionState, setActionState ] = useState( false );
+
+	const blockCommentId = useSelect( ( select ) => {
+		const clientID = select( blockEditorStore ).getSelectedBlockClientId();
+		return (
+			select( blockEditorStore ).getBlock( clientID )?.attributes
+				?.blockCommentId ?? false
+		);
+	}, [] );
+
+	const CommentBoard = ( { thread, parentThread } ) => {
+		return (
+			<>
+				<CommentHeader
+					thread={ thread }
+					onResolve={ () =>
+						setActionState( {
+							action: 'resolve',
+							id: parentThread?.id ?? thread.id,
+						} )
+					}
+					onEdit={ () =>
+						setActionState( { action: 'edit', id: thread.id } )
+					}
+					onDelete={ () =>
+						setActionState( { action: 'delete', id: thread.id } )
+					}
+					onReply={
+						! parentThread
+							? () =>
+									setActionState( {
+										action: 'reply',
+										id: thread.id,
+									} )
+							: undefined
+					}
+					status={ parentThread?.status ?? thread.status }
+				/>
+				<HStack
+					alignment="left"
+					spacing="3"
+					justify="flex-start"
+					className="editor-collab-sidebar-panel__user-comment"
+				>
+					<VStack
+						spacing="3"
+						className="editor-collab-sidebar-panel__comment-field"
+					>
+						{ 'edit' === actionState?.action &&
+							thread.id === actionState?.id && (
+								<EditComment
+									thread={ thread }
+									onUpdate={ ( value ) => {
+										onEditComment( thread.id, value );
+										setActionState( false );
+									} }
+									onCancel={ () => setActionState( false ) }
+								/>
+							) }
+						{ ( ! actionState ||
+							'edit' !== actionState?.action ) && (
+							<RawHTML>{ thread?.content?.rendered }</RawHTML>
+						) }
+					</VStack>
+				</HStack>
+			</>
+		);
+	};
+
+	return (
+		<>
+			{
+				// If there are no threads, show a message indicating no threads are available.
+				( ! Array.isArray( threads ) || threads.length === 0 ) && (
+					<VStack
+						alignment="left"
+						className="editor-collab-sidebar-panel__thread"
```

Comment:
Let's remove any classnames that aren't necessary.

Reply:
We applied this className for styling purposes and incorporated it into the SCSS file.